Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptAddView.jsp

```diff
@@ -178,24 +211,32 @@
 			html += '<td align="justify"><font size="2">' + description
 					+ '</font></td>';
 			html += '<td align="justify" hidden="true">' + id + '</td></tr>';
-
 			html += '</table>';
 			html += '<input type="hidden" name="syns" />';
 			addedsynonym += id;
+			if (onLoad == false) {
+				var t = $('#term').val();
+				t = term;
```

Comment:
why do you set t first and then override it with something else? What's the point? Same is true for the next two properties.